plaso/analysis/windows_services.py

```diff
@@ -263,11 +264,15 @@ def ExamineEvent(self, mediator, event, event_data):
       event (EventObject): event to examine.
       event_data (EventData): event data.
     """
-    if event_data.data_type not in self._SUPPORTED_EVENT_DATA_TYPES:
+    # TODO: Handle event log entries here also (ie, event id 4697).
+    if event_data.data_type != 'windows:registry:service':
       return
 
-    # TODO: Handle event log entries here also (ie, event id 4697).
-    service = WindowsService.FromEventData(event_data)
+    event_data_attributes = event_data.CopyToDict()
```

Comment:
Why is the "cast to WindowsRegistryServiceEventData" needed since you now use getattr int FromEventData ?

Reply:
@Onager please provide input